### PR TITLE
Test against PHP 7.2

### PR DESCRIPTION
**Pull request by joaorobertopb**
[ PHP 7.2](http://php.net/archive/2017.php#id2017-11-30-1) is now stable...

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -2,6 +2,7 @@ language: php
 
 php:
   - 7.1
+  - 7.2
```
Comment (nunomaduro) [inline .travis.yml]: Shouldn't we leave it anyway?

Comment (nunomaduro) [reply .travis.yml]: cc @joaorobertopb

Comment (joaorobertopb) [reply .travis.yml]: Humm ... In my opinion it would be interesting to have the tests run against php7.2 once it is stable.

**Comment by nunomaduro** [comment]
Thanks! 👍

**Pull request merged**